common/scylla_image_setup

```diff
@@ -24,7 +24,9 @@ from subprocess import run
 if __name__ == '__main__':
     cloud_instance = get_cloud_instance()
     run('/opt/scylladb/scylla-machine-image/scylla_configure.py', shell=True, check=True)
-    run('/opt/scylladb/scripts/scylla_sysconfig_setup --nic eth0 --setup-nic --ami', shell=True, check=True)
+# XXX: due to perftune.py bug, setup-nic crashes scylla_prepare, so disable it on Ubuntu for now
```

Comment:
so you skip `scylla_sysconfig_setup` ? not just the parameter causing the issue ?

Reply:
The issue was https://github.com/scylladb/scylla/issues/8041, but probably fixed now. I'll check it again

Reply:
Verified the issue fixed now, dropping this code